coordinator/src/node.rs

```diff
@@ -83,7 +93,9 @@ impl Node {
     }
 
     pub async fn update_settings(&self, settings: NodeSettings) {
-        *self.settings.write().await = settings;
+        let wallet_settings = settings.as_wallet_settings();
+        *self.settings.write().await = settings.clone();
+        self.inner.wallet().update_settings(wallet_settings).await;
```

Comment:
we actually are updating both settings (line 97 updates `NodeSettings).

I see how it can be confusing, as I tried to remove an unnecessary `clone()` which sacrificed readability 😔

will rewrite it.